src/Mutator/Removal/FunctionRemoval.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\Removal;
+
+use Infection\Mutator\Util\Mutator;
+use PhpParser\Node;
+
+/**
+ * @internal
+ */
+final class FunctionRemoval extends Mutator
```

Comment:
I suggest to rename these mutators:

* `FunctionRemoval` -> `FunctionCallRemoval`
* `MethodRemoval` -> `MethodCallRemoval`

as it better describes the intent of these methods and in line with the PHP-Parses node names (`FuncCall`, `MethodCall`)